### DISCUSSION TITLE
Starting Groot2 from command line

### DISCUSSION THREAD

**Discussion by katkunora** [BehaviorTree/BehaviorTree.CPP#881] [Q&A]
I'd like to start Groot2 from command line with options. I'd like Groot to start with the Monitor tab open and already open for connections. So in this when I click on a "Monitoring" button in my application, it opens up Groot and monitors the behaviortree immediately. Currently I need to start Groot first and click on "Connect" manually, I'd like to avoid that. 
Is there any way to do it?
Thank you

**Comment by Timple** [comment]
Try this branch: https://github.com/BehaviorTree/Groot/pull/187 ?
